content/graphqlschema/8.md

```diff
@@ -10,13 +10,14 @@ completed = 1
 type = "graphql"
 +++
 
-This is the end of the schema module.
+Congratulations on completing the schema module.
 
 The Dgraph instance you have running for the tutorial and the lessons in the
 modules so far give you a sandbox to play in. Make whatever queries or changes
 you like until you are comfortable with the material covered so far.
 
-<!-- TODO: Decide where to go next -->
+<!-- TBD: Decide where to go next -->
```

Comment:
This can be removed as already discussed and handled

Reply:
Fixed